curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md

```diff
@@ -39,6 +39,28 @@ assert(
 );
 ```
 
+Your Binary Search Tree should return false when checked with `isBinarySearchTree()`.
+
+```js
+assert(
+  (function () {
+    var test = false;
+    if (typeof BinarySearchTree !== 'undefined') {
+      test = new BinarySearchTree();
+    } else {
+      return false;
+    }
+    test.push(1);
+    test.push(5);
+    test.push(3);
+    test.push(2);
+    test.push(4);
+    test.root.left = new Node(1);
```

Comment:
Adding more elements might be not needed, as root node won't be fulfilling constrains.